### PR TITLE
Applicants can now update and delete their grants

### PR DESCRIPTION
**Pull request by danicavaldez**
# Description

Applicants can now successfully edit and delete grants after the backend update to make the grants array return with profile details. More work to come to clean up grants actions/reducers and profile actions/reducers.

### CODE REVIEW

File: src/store/actions/profileActions.js [hunk 1]
```diff
@@ -17,14 +17,6 @@ export const PUT_PROFILE_SUCCESS=
...
 "PUT_PROFILE_SUCCESS";
 
-// export const POST_GRANTS_START = "POST_GRANTS_START";
-// export const POST_GRANTS_SUCCESS = "POST_GRANTS_SUCCESS";
-// export const POST_GRANTS_FAILURE = "POST_GRANTS_FAILURE";
-
-// export const PUT_GRANTS_START = "GET_PROFILE_START";
-// export const PUT_GRANTS_SUCCESS = "GET_PROFILE_SUCCESS";
-// export const PUT_GRANTS_FAILURE = "GET_PROFILE_FAILURE";
-
```
Comment (LeeWood) [inline src/store/actions/profileActions.js:27]: Thanks for cleaning up commented code!

File: src/store/reducers/grantsReducer.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ import {
 
 const initialState = {
   grants: [],
-  profileGrants: [],
+  applicantGrants: [],
```
Comment (LeeWood) [inline src/store/reducers/grantsReducer.js:21]: I think this name change will help with the readability and clarity, good choice!

**Comment by bwkarr77** [comment]
Looks like a good solution. And implementing the new backend stuff

File: src/components/applicant-profile/applicant-grants/UpdateGrant.js [hunk 3]
```diff
@@ -49,8 +53,9 @@ export default function GrantsForm() {
   const handleSubmit = async e => {
     e.preventDefault();
     dispatch(putGrants(id, grant));
-    dispatch(getGrantsByApplicantId(applicant_id));
+    dispatch(getGrantsByApplicantId(applicant_id))
     await history.push("/GrantsList");
+    console.log(grant, id)
```
Comment (LeeWood) [inline src/components/applicant-profile/applicant-grants/UpdateGrant.js:58]: Do we need this console log?

**Review by LeeWood — APPROVED** [review]
Nice work! I'm glad we're moving along with our grant functionality!

**Pull request merged**